staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -48,23 +49,23 @@ var (
 				4, 5, 6, 8, 10, 15, 20, 30, 45, 60},
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		[]string{"operation", "type"},
+		[]string{"operation", "group", "resource"},
```

Comment:
And here
@dgrisonnet

Reply:
also fine by me as long as it is in the changelog

Reply:
SG